rebar.lock

```diff
@@ -22,22 +22,19 @@
       {ref,"3ef5b48a028bb66f82b452c98ae515903096641c"}},
   1},
  {<<"enacl">>,
-  {git,"https://github.com/jlouis/enacl.git",
-      {ref,"885662c069ea06ed1dc9567b7fb68aa755a9505c"}},
+  {git,"https://github.com/aeternity/enacl.git",
+      {ref,"26180f42c0b3a450905d2efd8bc7fd5fd9cece75"}},
   0},
  {<<"enoise">>,{pkg,<<"enoise">>,<<"1.0.1">>},0},
  {<<"eper">>,{pkg,<<"eper">>,<<"0.99.1">>},0},
- {<<"erlexec">>,
-  {git,"https://github.com/saleyn/erlexec.git",
-      {ref,"97a5188da1d37c7d2f62e4589be6237394d40811"}},
-  0},
  {<<"exometer_core">>,{pkg,<<"exometer_core">>,<<"1.5.7">>},0},
  {<<"folsom">>,{pkg,<<"folsom">>,<<"0.8.7">>},1},
  {<<"goldrush">>,{pkg,<<"goldrush">>,<<"0.1.9">>},1},
  {<<"gproc">>,{pkg,<<"gproc">>,<<"0.6.1">>},0},
+ {<<"hex2bin">>,{pkg,<<"hex2bin">>,<<"1.0.0">>},1},
  {<<"hut">>,{pkg,<<"hut">>,<<"1.2.1">>},1},
  {<<"idna">>,
-  {git,"https://github.com/aeternity/erlang-idna",
+  {git,"https://github.com/benoitc/erlang-idna",
```

Comment:
What is our policy, do we want to have everything forked to our org? /cc @sennui

Reply:
At some point I just stopped harassing @sennui for repo permissions :-)

Generally I think it would be beneficial to have a org mirror of all dependencies and a process for managing those. But that is a bigger and separate endeavour.